deploy-disconnected-registry/olm-sync.sh

```diff
@@ -230,12 +232,173 @@ function mirror() {
     done
 }
 
+function mirror_certified() {
```

Comment:
In the future we will need to have a unique version of mirror function which should did the same thing but for different catalogs. We will need to create a new card

Reply:
I think this should be implemented with oc-mirror